module-4/backend/week_one.md

```diff
@@ -1,19 +1,38 @@
 ## Week One - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. What's the most useful thing you learned from completing the intermission week work?
+* I think for me it was just learning general JS syntax. I was finally able to look at JS and it made some sense. Before this, it just looked like gibberish.
+
 2. What are some tools to help debug JavaScript code?
+* Debugger and console.log on the front end. Pry on the backend.
+
 3. What are some tools you need in order to unit test your JavaScript?
+* Mocha and Chai.
+
 4. What is the syntax for invoking a function? Give an example.
+* Adding parens after the function name. doAThing() vs just doAThing
+
 5. What's `this` in JavaScript?
+* I'm still a little fuzzy on this, but as far as I understand it, it's the execution context you're currently in.
+
 6. What is Webpack and why is it useful?
+* Webpack is an asset handler. It compiles the code into a single file, including CSS, and serves it.
```

Comment:
Webpack isn't a server but `webpack-dev-server` is able to spin up a server. They are different though. Webpack is just the asset handler